pyaedt/application/MessageManager.py

```diff
@@ -343,44 +384,46 @@ def __init__(self, project_dir=None):
             self.logger = logging.getLogger(__name__)
 
     def add_error_message(self, message_text):
-        """Add a type 2 "Error" message to the logger
+        """Add a type 2 "Error" message to the log file.
 
         Parameters
         ----------
-        message_text :
-            message to show
+        message_text : str
+            Text to display as the message.
 
         Returns
         -------
+        str
+            Message text added to the log file.
 
         """
 
         self.logger.error(message_text)
 
     def add_warning_message(self, message_text):
-        """Add a "Warning" message to the logger
+        """Add a "Warning" message to the log file.
 
-        Parameters
-        ----------
-        message_text :
-            message to show
+        message_text : str
+            Text to display as the message.
 
         Returns
         -------
+        str
```

Comment:
No return here.  See how there's no decorator for the method?  That means that we're using the vanilla implementation of Python methods, meaning if there's no `return`, it's returning nothing (well, it's returning `None`, so that's something but you don't document it).  Please go through and update the other methods to account for this.